src/@ionic-native/plugins/intel-security/index.ts

```diff
@@ -67,17 +67,21 @@ export class IntelSecurity {
 /**
  * @hidden
  */
+@Plugin({
+  pluginName: 'IntelSecurity',
+  plugin: 'com-intel-security-cordova-plugin',
+  pluginRef: 'intel.security.secureData',
+  repo: ''
+})
 export class IntelSecurityData {
 
   /**
   * This creates a new instance of secure data using plain-text data.
   * @param id {string} Storage resource identifier.
   * @returns {Promise<any>} Returns a Promise that resolves with the instanceID of the created data instance, or rejects with an error.
   */
+  @Cordova()
```

Comment:
Since now we have the Cordova decorator, there is no need to write any functionally. We can just have the keyword `return` in there.

If the real method takes an object as a parameter instead of a string, then we need to change the parameter type in the method definition/docs.